config.py

```diff
@@ -0,0 +1,11 @@
+import os
+import logging
+from connection import get_database_uri
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+SQLALCHEMY_DATABASE_URI = get_database_uri()
+SQLALCHEMY_TRACK_MODIFICATIONS = False
+
```

Comment:
what is SECRET_KEY = 'for dev only"

For my own knowledge.

Reply:
I couldn't say, this is an artifact from copying from the professor sample project.